### DISCUSSION TITLE
Removing recurring job from code still calling the old method

### DISCUSSION THREAD

**Discussion by charanjeet123** [jobrunr/jobrunr#470] [Q&A]
Hello All, 
I am using JobRunr in springboot in my project. I created one recurring job and it runs fine on its time. Now, when I deleted this job from code and created new recurring job (with new recurring id, new job name, new method signature), and restarted my application, the old recurring job is still present in the DB (MongoAtlas) and also running at its old scheduled time. But since, the code is not present in the application, it is throwing following exception every scheduled time:

2022-05-25 11:39:46|ERROR|backgroundjob-worker-pool-3-thread-13|BackgroundJobPerformer.java|updateJobStateToFailedAndRunJobFilters|Job(id=c36fcdab-f3c8-46d3-99c3-92326f141cf5, jobName='Job1') processing failed: Job method not found
org.jobrunr.scheduling.exceptions.JobMethodNotFoundException: com.xxx.xxx.service.JobRunrServiceImpl.doJob1()

Using JobRunr - 5.1.2

Is there a bug?

Note: After deleting the old recurring job from JobRunr Dashboard, everything is working fine. So, on removing any job from code, do we have to manually remove that job from dashboard or from DB collection? Any way of happening this automatically?

**Comment by rdehuyss** [comment]
Hhmm, it looks like the delete of the recurring job did not succeed for some reason. Can you create a github repo that reproduces this?
